src/Common/tests/gtest_event_notifier.cpp

```diff
@@ -47,6 +47,4 @@ TEST(EventNotifier, SimpleTest)
     handler13.reset();
     EventNotifier::instance().notify(Coordination::Error::ZSESSIONEXPIRED);
     ASSERT_EQ(result, 1);
-
-    EventNotifier::shutdown();
```

Comment:
Do we really need this?

Reply:
Another unit test may use something that relies on event notifier, e.g. testkeeper https://github.com/ClickHouse/ClickHouse/blob/471de5e481e7be934643afd3c14bb85bdbbd7768/src/Common/ZooKeeper/ZooKeeperImpl.cpp#L1028

Since all tests executed in the same process, `EventNotifier::instance()` became invalid after finishing this test